app/jobs/push_priority_appeals_to_judges_job.rb

```diff
@@ -147,4 +150,14 @@ def priority_distributions_this_month_for_all_judges
   def priority_distributions_this_month
     Distribution.priority_pushed.completed.where(completed_at: 30.days.ago..Time.zone.now)
   end
+
+  def warm_veteran_attribs_for_priority_distributions
+    start_time = Time.zone.now
+    non_genpop_distro_case_ids = DistributedCase.where(distribution_id: @tied_distributions.pluck(:id)).pluck(:case_id)
+    veterans_refreshed = Veteran.update_veteran_attribs_service.update_veterans_for_appeals(non_genpop_distro_case_ids)
+    genpop_distro_case_ids = DistributedCase.where(distribution_id: @genpop_distributions.pluck(:id)).pluck(:case_id)
+    veterans_refreshed += Veteran.update_veteran_attribs_service.update_veterans_for_appeals(genpop_distro_case_ids)
```

Comment:
Let's minimize the number of DB queries:
```suggestion
    distro_case_ids = DistributedCase.where(distribution_id: (@tied_distributions + @genpop_distributions).pluck(:id)).pluck(:case_id)
    veterans_refreshed = Update_veteran_attribs_service.update_for_appeals(distro_case_ids)
```